examples/next/pages/ui/components/authenticator/sign-in-with-phone/index.page.tsx

```diff
@@ -20,8 +20,13 @@ const formFields = {
   },
 };
 
-function App({ signOut }) {
-  return <button onClick={signOut}>Sign out</button>;
+function App({ user, signOut }) {
+  return (
+    <>
+      <h2>Welcome, {user.attributes?.phone_number}</h2>
```

Comment:
nit: so it would be "welcome, 1234567"? 🤔   would it be better to use `username`, but it's "sign-in-with-phone", so ... just a thought

Reply:
Yep, for backend with phone number only, usernames are often filled with dummy data, so I'm testing for phone number here. Other tests like `sign-in-with-username` will test for username 🙏